src/overlays/actors/ovl_Bg_Jya_1flift/z_bg_jya_1flift.h

```diff
@@ -6,9 +6,16 @@
 
 struct BgJya1flift;
 
+typedef void (*BgJya1fliftActionFunc)(struct BgJya1flift*, GlobalContext*);
+
 typedef struct BgJya1flift {
-    /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x70];
+    /* 0x0000 */ DynaPolyActor dyna;
+    /* 0x0164 */ ColliderCylinder collider;
+    /* 0x01B0 */ BgJya1fliftActionFunc actionFunc;
+    /* 0x01B4 */ s16 moveDelay;
+    /* 0x01B6 */ u8 direction; // 1 is moving down and 0 is moving up
```

Comment:
maybe this variable can be called isGoingDown, that way true is down and false is up (and this comment wont be needed anymore)